instances/widgets.treasury-factory.near/widget/pages/settings/index.jsx

```diff
@@ -3,7 +3,7 @@ const { instance, tab } = props;
 const [leftNavbarOptions, setLeftBarOptions] = useState([
   {
     title: "Pending Requests",
-    key: "requests",
+    key: "pending-requests",
```

Comment:
this will change the endpoint URL? 
if yes please make sure we don't use the older endpoint anywhere else.

also if this fixes the issue with the pending table loader, we should apply it to other pages as well.

Reply:
yes it fixes issue with loading when we switching tabs.